### PR TITLE
Possible typo for BGN/PCGN Japanese system 1976?

### PR DESCRIPTION
**Pull request by ribose-jeffreylau**
The file name says 1976 but its content says 1930.  Which one is correct?

### CODE REVIEW

**Comment by ronaldtse** [comment]
@ribose-jeffreylau 1976 and 1930 are the same system. From the current document:

> The modified Hepburn system for the romanization of Japanese has been in use by the BGN and the PCGN since the 1930’s and has been used extensively in the romanization of Japanese geographic names. This system is well adapted to the general needs of speakers of English and is the most widely used system for romanization of Japanese. In 2007, the Geospatial Information Authority of Japan (GSI) issued “Toponymic Guidelines for Map Editors and other Editors, Japan (Third Edition)2,” and the BGN and the PCGN have adopted several of the guidelines dealing with romanization.

I think:
* 1930 modified Hepburn system used by BGN and PCGN (BGN/PCGN joint Romanization systems started in 1947: [history](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/837836/BGN_PCGN_Relationship_2019.pdf))

* 1976 there was a BGN/PCGN Agreement for Romanization of Japanese

* 2017 they adopted rules from GSI's "Toponymic Guidelines for Map Editors and other Editors, Japan (Third Edition)" and this provides the "Notes" section about how to transliterate geonames.

In summary I believe these are the "systems":
* `bgn-jpn-Hrkt-Latn-1930`, fully inherits modified Hepburn (don't know the year of the PCGN system)
* `bgnpcgn-jpn-Hrkt-Latn-1976`, fully inherits modified Hepburn
* `bgnpcgn-jpn-Hrkt-Latn-2017` fully inherits modified Hepburn + new rules.

(use the `inherits` keyword, see other systems)

Can you help address them this way in this repo and in the document? Thanks!

**Comment by ronaldtse** [comment]
These codes have been addressed in https://gitlab.com/calconnect/tc-localization/csd-transcription-systems/commit/78bb7c4aa4c9e7a8f9716815a128f92a413463f1.

We just need to update the implementations here.

**Comment by ribose-jeffreylau** [comment]
Thanks for clarifying.  That makes this PR obsolete.  Closing.
